src/main/java/fr/minuskube/inv/ClickableItem.java

```diff
@@ -16,29 +16,41 @@
 
 package fr.minuskube.inv;
 
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+import org.bukkit.Material;
 import org.bukkit.entity.Player;
 import org.bukkit.event.inventory.InventoryClickEvent;
 import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Adding JetBrains annotations isn't a Clean-Up. Its adding JetBrains annotations and should be separate